src/jdbc/cubrid/jdbc/driver/CUBRIDServerSidePreparedStatement.java

```diff
@@ -1,31 +1,31 @@
 /*
- * Copyright (C) 2008 Search Solution Corporation
- * Copyright (C) 2016 CUBRID Corporation
+ * Copyright (C) 2008 Search Solution Corporation. 
```

Comment:
This file is newly create 8 months ago in CUBRID.
Copyright 2008 Search Solution is necessary ?

Reply:
Thank you, I will remove a copyright of Search Solution at comment.